### PR TITLE
add notice on credit refresh monthly

### PR DESCRIPTION
**Pull request by Vincexodus**
### Description

add credit refresh notice below on credit progress bar 

### Related Issue

#283

### Changes Made

add `* Credits refresh every month` text beside credit percentage
removed text-size styling typo

### Screenshots

Screenshot 2024-09-15 172709

### Checklist

- [x ] I have tagged the issue in this PR.
- [x ] I have attached necessary screenshots.
- [x ] I have provided a short description of the PR.
- [x ] I ran `yarn build` and build is successful
- [x ] My code follows the style guidelines of this project.
- [x ] I have added necessary documentation (if applicable)

### CODE REVIEW

**Comment by Vincexodus** [comment]
@Fryingpannn I feel like the row of text `Current Plan     Free Trial` is abit redundant since top right badge has shown the same thing. Do u need me to remove it? I can amend this pr

**Comment by eiddor** [comment]
Not sure how much work this would be, but the annual plan view has a "Current Period" makes sense for this since it's logically related to the credit usage display and users will know exactly when their credits will refresh.

Then there can be a "Membership Period" that reflects the membership's timeframe (more useful for annuals)

image

File: components/dashboard/freetrial-card.tsx [hunk 1]
```diff
@@ -57,15 +57,20 @@ export default function FreeTrialCard({
...
-              {loading ? "-" : usage.percent_credit_used}% of free trial PearAI
-              Credits used
-            </p>
+            <div className="flex justify-between">
+              <p className="text-sm text-muted-foreground">
+                {loading ? "-" : usage.percent_credit_used}% of free trial
+                PearAI Credits used
+              </p>
+              <p className="text-right text-sm text-muted-foreground">
+                * Credits refresh every month
```
Comment (Fryingpannn) [inline components/dashboard/freetrial-card.tsx:66]:
```suggestion
                Credits refresh every month
```

**Review by Fryingpannn — DISMISSED** [review]
Ty!

**Review by Fryingpannn — APPROVED** [review]
ty!

**Pull request merged**